### DISCUSSION TITLE
Use Pretrained model from MONAI Label in MONAI Core

### DISCUSSION THREAD

**Discussion by kmanuwai** [Project-MONAI/MONAI#6014] [Q&A]
I have images labeled and a DynUNet (Deepedit) model trained through the MONAI Label process. Now I would like to take this trained model and further train it on MONAI Core in a Jupyter Notebook like this one: https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/spleen_segmentation_3d.ipynb 

1. Is this possible?
2. It seems like the trained deepdit model from MONAI Label has inputs such as clicks. So is using a model trained with deepedit not trainable in MONAI Core? Is only the Automated Segmentation Annotation Paradigm able to be transferred? (see https://github.com/Project-MONAI/MONAILabel/wiki/Annotation-Paradigms)

**Comment by KumoLiu** [comment]
Hi @tangy5, could you please share some comments here? Thanks in advance!

**Reply by wyli** [reply]
in case you missed it, there's a deepedit tutorial https://github.com/Project-MONAI/tutorials/tree/main/deepedit/ignite
